kolibri/plugins/coach/assets/src/views/plan/LessonEditDetailsPage/index.vue

```diff
@@ -48,7 +49,7 @@
       CoachImmersivePage,
       ResourceListTable,
     },
-    mixins: [coachStringsMixin, commonCoreStrings],
+    mixins: [commonCoach, coachStringsMixin, commonCoreStrings],
```

Comment:
This is a lot to add to the component simply to access the single isSuperuser - would be better to use the composable directly here.